recipes/pure_eval/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "pure_eval" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 207facfb53f183e740be37d3e13f25968e16e7799d7725940b3ee008266903c1
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+  skip: true  # [py<35]
```

Comment:
I think this recipe can be `noarch`. In that case, please remove the `skip` line, and use `python >=3.5` in both `host` and `run` sections below.